lib/gamco.ex

```diff
@@ -3,16 +3,39 @@ defmodule Gamco do
   Documentation for `Gamco`.
   """
 
-  @doc """
-  Hello world.
+  defmacro __using__(opts) do
+    quote bind_quoted: [opts: opts] do
+      alias Gamco.{Secure, Builder}
 
-  ## Examples
+      @otp_app Keyword.fetch!(opts, :otp_app)
 
-      iex> Gamco.hello()
-      :world
+      def ga_javascript_tags(opts \\ []) do
+        tag_id = Keyword.get(opts, :tag_id, ga_tag_id())
+        if ga_active?(), do: Builder.javascript_tags(tag_id, opts)
```

Comment:
do you think using with is more readable?

```
def ga_javascript_tags(opts \\ []) do
  with true <- ga_active?() do
    opts
    |> Keyword.get(:tag_id, ga_tag_id())
    |> Builder.javascript_tags(opts)
  else
    _ -> nil
  end
end
```

Reply:
Mmm I do not know, what about a multi line if block, I think it will be even more readable?

```elixir
def ga_javascript_tags(opts \\ []) do
  if ga_active?() do
    opts
    |> Keyword.get(:tag_id, ga_tag_id())
    |> Builder.javascript_tags(opts)
  end
end
```

Reply:
By now using if feels like an anti pattern :-)

but yeah I like this better than the original, but also I like the with as the return nil is explicit and not something you have to figure it out on your own